### ISSUE TITLE
multibox loss

### ISSUE THREAD

**Issue by XDUSPONGE** [yxlijun/S3FD.pytorch#27]
Thanks for your code
in the paper
* the cls loss is normalized by the number of positive and negative anchors
* the reg term is normalized by the number of positive anchors.
so may be
![image](https://user-images.githubusercontent.com/11554033/75959732-bd0f8500-5ef9-11ea-923d-c0caccda81c9.png)
it should be changed
```python
M = num_pos.data.sum().float() + num_neg.data.sum().float()
loss_c /= N
```
also in the paper `L =\lambda * L_{cls} + L_{reg}`
in the code
```python
loss_l, loss_c = criterion(out, targets)
loss = loss_l + loss_c
```
may be the two place should be modified
